_posts/2020-06-01-Fortran-Newsletter-June-2020.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: "Fortran newsletter: June 2020"
+category: newsletter
+author: Milan Curcic
+---
+
+```fortran
+print *, 'Hello, World!'
+```
```

Comment:
I thought this was cute for the very first newsletter, but perhaps unnecessary to repeat in each.